packages/cli/commands/auth.js

```diff
@@ -76,15 +76,15 @@ exports.handler = async options => {
   setLogLevel(options);
   logDebugInfo(options);
 
+  const env = qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion(getConfigPath());
+
   if (!getConfigPath()) {
```

Comment:
Curious why this was moved. Does it still makes sense to check for a config path after trying to load the config?

Reply:
The `loadConfig` function calls `CLIConfiguration.init()`, which then sets `CLIConfiguration.isActive()`, which alerts us that the customer is using the new config. Had to move it up in the file, so that `getConfigPath()` works.

Reply:
If we use the flag idea @brandenrodgers suggested, can we make a call to `CLIConfiguration.init()` directly if that flag is provided?